tests/PathIdentificationTest.php

```diff
@@ -123,3 +127,17 @@
         ->withoutExceptionHandling()
         ->get('/acme/foo/abc/xyz');
 });
+
+test('tenant set as a default parameter for the URLs', function () {
+    Tenant::create([
+        'id' => 'acme',
+    ]);
+
+    expect(tenancy()->initialized)->toBeFalse();
+
+    pest()->get(route('foo', ['tenant' => 'acme', 'a' => 1, 'b' => 2]));
+
+    expect(tenancy()->initialized)->toBeTrue();
+    expect(tenant('id'))->toBe('acme');
+    pest()->get(route('baz', ['a' => 1, 'b' => 2]))->assertOk(); // Assert route don't need tenant parameter
```

Comment:
Also assert that the generated route is equal to what the value would be if we did specify the `tenant` argument.